d2/src/main/java/com/linkedin/d2/balancer/dualread/DualReadStateManager.java

```diff
@@ -79,19 +79,19 @@ public DualReadStateManager(DualReadModeProvider dualReadModeProvider, Scheduled
   public void updateGlobal(DualReadModeProvider.DualReadMode mode)
   {
     _dualReadMode = mode;
-    LOG.debug("Global dual read mode updated: " + mode);
+    LOG.info("Global dual read mode updated: " + mode);
```

Comment:
While you have this file open, can you update this to use `{}` formatting instead of `+`?